versions.yaml

```diff
@@ -64,10 +64,10 @@ module-sets:
       - go.opentelemetry.io/contrib/samplers/jaegerremote
       - go.opentelemetry.io/contrib/samplers/jaegerremote/example
       - go.opentelemetry.io/contrib/samplers/probability/consistent
-  experimental-config:
+  experimental-otelconf:
     version: v0.14.0
     modules:
-      - go.opentelemetry.io/contrib/config
+      - go.opentelemetry.io/contrib/otelconf
```

Comment:
You need to keep config here, or the deprecation won't be released.

Reply:
This looks updated. Resolving.